src/perf/lib/RpsClient.cpp

```diff
@@ -136,10 +136,17 @@ RpsClient::Start(
     }
 
     QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
+    uint32_t ActiveProcCount = QuicProcActiveCount();
```

Comment:
I think you should only need to leave 1 core free; specifically core 0 (so add one to the index below).

Reply:
Actually, now I realize you're trying to explicitly leave cores free in the second group. But one should still be enough.

Reply:
The reason I went 2 was because of hyperthreading. Usually the hyperthread core is the 2nd one in a pair. So pulling 2 in general removes an entire physical core and it's hyperthread pair

Reply:
Ok.